proto/cosmos/base/reflection/v1beta1/reflection.proto

```diff
@@ -3,10 +3,14 @@ package cosmos.base.reflection.v1beta1;
 
 import "google/api/annotations.proto";
 
-option go_package = "github.com/cosmos/cosmos-sdk/client/grpc/reflection";
+option go_package = "github.com/cosmos/cosmos-sdk/server/grpc/appreflection";
 
-// ReflectionService defines a service for interface reflection.
+// ReflectionService defines a service for application reflection.
 service ReflectionService {
+  // GetAppDescriptor returns the full cosmos application descriptor
+  rpc GetAppDescriptor(GetAppDescriptorRequest) returns (GetAppDescriptorResponse) {
```

Comment:
Rather than a single `AppDescriptor` I would personally prefer individual methods for each type of information. Not a strongly held preference, but to me that would feel cleaner.

Reply:
I thought about this too, but when experimenting in the end you'd need to get sequentially all this descriptor to be able to  write on a chain, but we can split them.